client/src/img/splash.svg

```diff
@@ -0,0 +1,10213 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Removed. I'm considering using them for the splash screen on the Pi while it boots. Probably don't belong here though.